src/nvim/generators/gen_eval.lua

```diff
@@ -62,7 +62,8 @@ for name, def in pairs(funcs) do
   local base = def.base or "BASE_NONE"
   local func = def.func or ('f_' .. name)
   local data = def.data or "NULL"
-  gperfpipe:write(('%s,  %s, %s, %s, &%s, (FunPtr)%s\n')
-                  :format(name, args[1], args[2], base, func, data))
+  local fast = def.fast or false
+  gperfpipe:write(('%s,  %s, %s, %s, %s, &%s, (FunPtr)%s\n')
+                  :format(name, args[1], args[2], base, fast, func, data))
```

Comment:
based on line 45:

    %define initializer-suffix ,0,0,false,BASE_NONE,NULL,NULL

I would expect `fast` to be in the 4th-last place:
```suggestion
                  :format(name, args[1], args[2], fast, base, func, data))
```

Reply:
There is a mistake in the initializer. My guess is that BASE_NONE is 0 so it just happens to work.